pkg/patterns/declarative/reconciler.go

```diff
@@ -173,6 +183,43 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 		return reconcile.Result{}, fmt.Errorf("error applying manifest: %v", err)
 	}
 
+	statusMap := make(map[status.Status]bool)
+	for _, object := range objects.Items {
+
+		unstruct, err := getObjectFromCluster(object, r.config, r.dynamicClient)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+		res, err := status.Compute(unstruct)
+		if err != nil {
+			log.WithValues("unstuctured", unstruct).Error(err, "Unable to get status of unstructured")
+			statusMap[status.NotFoundStatus] = true
+		}
+		if res != nil {
+			log.WithValues("kind", object.Kind).WithValues("name", object.Name).WithValues("status", res.Status).WithValues(
```

Comment:
This might get a little verbose - we can try it and see!

Reply:
Should we just log the `kind` and `name` without the `status`

Reply:
It's just that you're going to be logging for every "child" object each time you go round the reconcile loop.  I think it's fine to leave in for now, because it will help us understand what's going on, and it's also going to be very obvious; we'll just likely want to reduce it to V(2) once we're all happy that it works as we expect!